src/ejabberd_c2s.erl

```diff
@@ -3350,3 +3380,9 @@ setup_accum(Acc, StateData) ->
     Server = StateData#state.server,
     mongoose_acc:update(Acc, #{server => Server, user => User}).
 
+%% @doc This function is executed when c2s receives a stanza from TCP connection.
+-spec from_sender_element(jlib:xmlel(), StateData :: state()) -> mongoose_acc:t().
+from_sender_element(El, #state{sid = SID, jid = JID}) ->
```

Comment:
Maybe, a better name for this function would be sth like: `acc_from_element_with_sender`

Reply:
`element_to_sender_acc`. Because we use to in `list_to_binary`-family of functions.

p.s.: I wanted to use `element_to_outgoing_acc`, but I have problems with outgoing and incoming atoms in our code. There is `incoming` message from a client connection socket. But `outgoing` message from c2s to be routed.